src/coprocessor/dag/expr/builtin_time.rs

```diff
@@ -311,4 +340,25 @@ mod test {
             assert_eq!(got, exp);
         }
     }
+
+    #[test]
+    fn test_year_week() {
+        let tests = vec![
+            ("2011-11-11", 201145i64),
+            ("2008-02-10", 200806i64),
+            ("2000-02-11", 200006i64),
+            ("2100-02-11", 210006i64),
+            ("2011-11-11 10:10:10", 201145i64),
```

Comment:
Test data in `test_last_day` are created specifically for testing a month's last day, you may cover edge cases specific to `year_week` (like first/last day of some weeks).